ext/SymPyPythonCallTermInterfaceExt.jl

```diff
@@ -0,0 +1,52 @@
+module SymPyPythonCallTermInterfaceExt
+
+import SymPyPythonCall
+import TermInterface
```

Comment:
TermInterface was effectively deprecated. Doing this directly on the terms in SymbolicUtils would be more helpful.

Reply:
Thanks for that detail!